packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/Head/generated_samples/http_success_head200.py

```diff
@@ -29,7 +29,7 @@ def main():
     )
 
     response = client.http_success.head200()
-    print(response)
+    print(response.serialize())
```

Comment:
we wouldn't want the serialized version of the response, that's what we're sending to the service. we could print `response.content` or `response.json()`

Reply:
let us use `to_dict()` as discussed.